virtual_rainforest/soil/carbon.py

```diff
@@ -0,0 +1,203 @@
+"""The `soil.carbon` module.
```

Comment:
Yeah we did agree that that nesting would be better. I didn't change it because the nesting has been changed in a different pull request and I was concerned about merge conflicts, but is it more sensible to just change it wherever appropriate and let git handle the merging?